Makefile

```diff
@@ -77,7 +77,8 @@ toolset: $(TOOLSET_PATH)
 $(TOOLSET_PATH): $(BINARIES_PATH)/dotnet-cli
 	pushd $(TOOLSET_SRC_PATH) ; \
 	$(BINARIES_PATH)/dotnet-cli/dotnet restore && \
-	$(BINARIES_PATH)/dotnet-cli/dotnet publish -o $(TOOLSET_PATH)
+	$(BINARIES_PATH)/dotnet-cli/dotnet publish -o $(TOOLSET_PATH) && \
+	sed -i -e 's/Microsoft.CSharp.Targets/Microsoft.CSharp.targets/g' $(TOOLSET_PATH)/Extensions/Microsoft/Portable/v5.0/Microsoft.Portable.CSharp.targets
```

Comment:
Seems like we should fix.  Is there a tracking bug on us or the appropriate owner?